Patterns/Turkish/Turkish-Numbers.yaml

```diff
@@ -0,0 +1,455 @@
+  references: [HundredsNumberIntegerRegex, OneToNineDenominatorSuffixRegex, TenToHundredDenominatorSuffixRegex, HundredsDenominatorSuffixRegex]
+ThousandToMillionDenominatorSuffixRegex: !nestedRegex
+  def: ({ThousandsNumberIntegerRegex}((\s{HundredsNumberIntegerRegex})?(\s{TensNumberIntegerRegex})?\s{OneToNineDenominatorSuffixRegex}|{TenToHundredDenominatorSuffixRegex}|{HundredToThousandDenominatorSuffixRegex})|{ThousandsDenominatorSuffixRegex})
+  references: [ThousandsNumberIntegerRegex, HundredsNumberIntegerRegex, TensNumberIntegerRegex, OneToNineDenominatorSuffixRegex, TenToHundredDenominatorSuffixRegex, HundredToThousandDenominatorSuffixRegex, ThousandsDenominatorSuffixRegex]
+MillionToBillionDenominatorSuffixRegex: !nestedRegex
+  def: ({MillionsNumberIntegerRegex}((\s{ThousandsNumberIntegerRegex})?(\s{HundredsNumberIntegerRegex})?(\s{TensNumberIntegerRegex})?\s{OneToNineDenominatorSuffixRegex}|{TenToHundredDenominatorSuffixRegex}|{HundredToThousandDenominatorSuffixRegex}|{ThousandToMillionDenominatorSuffixRegex})|{MillionsDenominatorSuffixRegex})
+  references: [MillionsNumberIntegerRegex, ThousandsNumberIntegerRegex, HundredsNumberIntegerRegex, TensNumberIntegerRegex, OneToNineDenominatorSuffixRegex, TenToHundredDenominatorSuffixRegex, HundredToThousandDenominatorSuffixRegex, ThousandToMillionDenominatorSuffixRegex, MillionsDenominatorSuffixRegex]
+BillionToTrillionDenominatorSuffixRegex: !nestedRegex
+  def: ({BillionsNumberIntegerRegex}((\s{MillionsNumberIntegerRegex})?(\s{ThousandsNumberIntegerRegex})?(\s{HundredsNumberIntegerRegex})?(\s{TensNumberIntegerRegex})?\s{OneToNineDenominatorSuffixRegex}|{TenToHundredDenominatorSuffixRegex}|{HundredToThousandDenominatorSuffixRegex}|{ThousandToMillionDenominatorSuffixRegex}|{MillionToBillionDenominatorSuffixRegex})|{BillionsDenominatorSuffixRegex})
+  references: [BillionsNumberIntegerRegex, MillionsNumberIntegerRegex, ThousandsNumberIntegerRegex, HundredsNumberIntegerRegex, TensNumberIntegerRegex, OneToNineDenominatorSuffixRegex, TenToHundredDenominatorSuffixRegex, HundredToThousandDenominatorSuffixRegex, ThousandToMillionDenominatorSuffixRegex, MillionToBillionDenominatorSuffixRegex, BillionsDenominatorSuffixRegex]
+AllDenominatorSuffixRegex: !nestedRegex
+  def: ({OneToNineDenominatorSuffixRegex}|{TenToHundredDenominatorSuffixRegex}|{HundredToThousandDenominatorSuffixRegex}|{ThousandToMillionDenominatorSuffixRegex}|{MillionToBillionDenominatorSuffixRegex}|{BillionToTrillionDenominatorSuffixRegex}|{RoundNumberDenominatorSuffixRegex})
+  references: [OneToNineDenominatorSuffixRegex, TenToHundredDenominatorSuffixRegex, HundredToThousandDenominatorSuffixRegex, ThousandToMillionDenominatorSuffixRegex, MillionToBillionDenominatorSuffixRegex, BillionToTrillionDenominatorSuffixRegex, RoundNumberDenominatorSuffixRegex ]
+FractionPrepositionRegex: !nestedRegex
+  def: (?<=\b)(?<numerator>({AllIntRegex})|((?<![\.,])\d+))\s+(bölü)\s+(?<denominator>({AllIntRegex})|(\d+)(?![\.,]))(?=\b)
```

Comment:
If Turkish always uses "," as decimal separator and "." for thousand separator, all occurrences of [\.,] should probably be changed to the decimal separator only "[\.,]" -> ",".

Reply:
noted, we will try this change.